src/databricks/labs/ucx/hive_metastore/workflows.py

```diff
@@ -27,13 +27,25 @@ def migrate_dbfs_root_delta_tables(self, ctx: RuntimeContext):
         ctx.tables_migrator.migrate_tables(what=What.DBFS_ROOT_DELTA, acl_strategy=[AclMigrationWhat.LEGACY_TACL])
 
 
-class MigrateTablesInMounts(Workflow):
+class ScanTablesInMounts(Workflow):
     def __init__(self):
-        super().__init__('migrate-tables-in-mounts-experimental')
+        super().__init__('scan-tables-in-mounts-experimental')
 
     @job_task
     def scan_tables_in_mounts_experimental(self, ctx: RuntimeContext):
         """[EXPERIMENTAL] This workflow scans for Delta tables inside all mount points
         captured during the assessment. It will store the results under the `tables` table
         located under the assessment."""
         ctx.tables_in_mounts.snapshot()
```

Comment:
Please integration test TablesInMounts as well.

Reply:
It's actually very hard to integration test TablesInMounts because `dbutils` acts differently when used in the SDK vs when used in a Databricks cluster.
For example, calling dbutils.fs.ls:
- returns dbfs:/mnt/xxx on SDK
- returns dbfs:/mnt/xxx/ on a cluster

This code is made to handle the behaviour of a Spark cluster as much as possible.

Additionally, some utility methods are missing in SDK, like isDir()

@nfx FYI